sdks/wasm/bindings-test.cs

```diff
@@ -42,10 +42,17 @@ public class TestClass {
 	}
 
 	public static int int_val;
-	public static void InvokeInt (int i) {
+    public static void InvokeInt(int i)
+	{
```

Comment:
Identation problem.

Reply:
Yes... Still battling with those. The convention in this file is not consistent, I'll make another PR to adjust it.